virtualize.sh

```diff
@@ -148,6 +148,12 @@ ssh $SSHOPTS root@$installserverip "
 
 kill ${tail_job}
 
+# Dump elasticsearch logs into /var/log/elasticdump,
+# upload_logs will update the dump in swift.
+scp $SSHOPTS $ORIG/dumpelastic.py root@$installserverip:/tmp
```

Comment:
I mean, Why don't we just run `./dumpelastic.py --url=http://$installserverip:9200` from the local machine?